src/Data/List/NonEmpty/Base.agda

```diff
@@ -85,6 +85,11 @@ map f (x ∷ xs) = (f x ∷ List.map f xs)
 replicate : ∀ n → n ≢ 0 → A → List⁺ A
 replicate n n≢0 a = a ∷ List.replicate (pred n) a
 
+drop : ℕ → List⁺ A → List⁺ A
+drop zero    xs           = xs
+drop (suc n) (x ∷ [])     = x ∷ []
```

Comment:
This makes sense, post facto, given the constraints, but is highly non-obvious given just the signature. I would add a comment both in the CHANGELOG and here about that.

Reply:
will add a comment.

Reply:
I'm not happy with drop returning a non-empty list like this. It doesn't feel right at all to me

Reply:
this is not a surprising handling:

https://github.com/agda/agda-stdlib/blob/e3533e8ccbef6ebc1558f79f12661f482e57d49e/src/Data/List/Base.agda#L271-L274

there can be another implementing using `Fin` but this implementation is handy in cases where extra dependent types are unwanted.